### ISSUE TITLE
Like button image is incorrectly linked, crashing VS

### ISSUE THREAD

**Issue by AmadeusW** [justcla/HotTips#43]
When I debug another extension than HotTips, VS crashes with the following message:

**System.IO.DirectoryNotFoundException: 'Could not find a part of the path 'C:\src\surround\Surround.VS\bin\Debug\Tips\images\Like.png'.'**

where `\Tips\images\Like.png` comes from the HotTips extension, and `C:\src\surround` is another VS extension. This leads me to believe that HotTips is trying to access "Like.png" from the arbitrary location of _currently active extension_, which HotTips is not when I'm not debugging it. The fix will likely entail correctly packaging and using the resources.

**Comment by justcla** [comment]
Deep.

**Comment by AmadeusW** [comment]
The code is in TipOfTheDayWindow.xaml.cs: `new BitmapImage(new Uri("Tips/images/Like.png", UriKind.Relative));` <- this tries to load a file from current running directory. Instead, the BitmapImage needs to load the resource. The images itself should *not* use the "copy to output directory" property since they will be embedded

**Comment by justcla** [comment]
I'm getting this when running in normal (non-debug) mode:

![image](https://user-images.githubusercontent.com/17131343/43284436-a2e30c7c-90d0-11e8-9f42-d50d233fe311.png)

**Comment by justcla** [comment]
@prafullbhosale Any ideas?

**Comment by AmadeusW** [comment]
yeah that's the issue. The bug is fixed through #50, although this is not the correct fix. Ultimately, we want to consume these images as assembly resources.
